Offboarding/logging_for_deleting_accounts.log

```diff
@@ -0,0 +1,11 @@
+INFO:root:                   Environment name of the account in CloudCheckr
+0  https://api.cloudcheckr.com                    test46534534545
+1  https://api.cloudcheckr.com                        testaccount
+INFO:root:Started deleting accounts
+INFO:root:https://api.cloudcheckr.com
```

Comment:
Don't think it needs to log the environment multiple times

Reply:
I think I removed the whole logging thing for this part, just going to log the API request parameters so each call is logged, whether pass or fail